foundry.toml

```diff
@@ -73,6 +74,7 @@ base = "${ETH_NODE_URI_BASE}"
 
 [etherscan]
 mainnet = { key = "${MAINNET_ETHERSCAN_API_KEY}", url = "https://api.etherscan.io/v2/api?chainid=1", chain = "1" }
+apechain = { key = "${APECHAIN_ETHERSCAN_API_KEY}", url = "https://api.apescan.io/api", chain = "33139" }
```

Comment:
I saw etherscan v2 supports Apechain: https://docs.etherscan.io/etherscan-v2/getting-started/supported-chains

Reply:
@ezynda3 why did you resolve this correct comment without any response and without a fix?

Reply:
@0xDEnYO I commented that it wasn't working and was getting an invalid API key error. The comment didn't save apparently.

Reply:
ok, I see. Thanks